torchdata/dataloader2/reading_service.py

```diff
@@ -369,7 +369,7 @@ class MultiProcessingReadingService(ReadingServiceInterface):
     pin_memory: bool
     timeout: float
     worker_init_fn: Optional[Callable[[int], None]]
-    prefetch_factor: int
+    prefetch_factor: Optional[int]
```

Comment:
noob question: why it's called "factor" ?

Reply:
Argument from DataLoader1
https://github.com/pytorch/pytorch/blob/837ca8f344380f2356b01662f215ff561b09401f/torch/utils/data/dataloader.py#L176

`prefetch_factor * num_workers`

Reply:
It doesn't have to be done in this PR; maybe add docsting for arguments in MultiProcessingReadingServicer? Thanks.

Reply:
> ``2`` means there will be a total of 2 * num_workers batches prefetched across all workers.

noob question: it sounds like different data loader workers may have different numbers of prefetched batches? -- because otherwise, the doc should just say "2 means each worker will prefetech 2 batches"

Reply:
For `DataLoader1`, each worker always prefetches `prefetch_factor` batches.